build

```diff
@@ -9,7 +9,7 @@ if [ ! -h gopath/src/${REPO_PATH} ]; then
 fi
 
 export GOBIN=${PWD}/bin
-export GOPATH=${PWD}/gopath
+export GOPATH=${PWD}/gopath:`pwd`/Godeps/_workspace
```

Comment:
Why Godeps first? Wouldn't you want code local to this repo to be searched before dependencies?